### ISSUE TITLE
[FEATURE REQUEST] Favorite IP's

### ISSUE THREAD

**Issue by HeroLeam** [rivafarabi/deckboard#1093]
**Is your feature request related to a problem? Please describe.**
Every day I need to enter different IPs, as I use them at home and at work, and as the IP changes I need to enter them manually every day, as it only saves the last IP used.

**Describe the solution you'd like**
Create something where you can save IPs as a shortcut in the main part of the app

**Describe alternatives you've considered**
is to save IP's on the tablet's keyboard for use, but it is still laborious because when you place it, it always adds a space at the end, but I still need to delete this space for the app to detect it as an IP.

**Additional context**
Being able to create buttons similar to this, where you can create several IP presets
![Screenshot_20240514-070841](https://github.com/rivafarabi/deckboard/assets/92167805/937f1628-a7ee-410a-8d41-9b0dfcae8253)

**Comment by schousda** [comment]
I also requested this feature:
https://github.com/rivafarabi/deckboard/issues/1089

Would be very nice, if this will be part of the app.

**Comment by Bus42** [comment]
Agreed. I run a multi-computer setup so it would help to be able to have my favorite IP addresses easily accessible.
